python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -89,6 +89,21 @@ class CalibrateImageConnections(pipeBase.PipelineTaskConnections,
         dimensions=["instrument", "exposure", "detector"],
     )
 
+    background_flat = connectionTypes.PrerequisiteInput(
+        name="flat",
+        doc="Flat calibration frame used for background correction.",
+        storageClass="Exposure",
+        dimensions=["instrument", "detector", "physical_filter"],
+        isCalibration=True,
+    )
+    illumination_correction = connectionTypes.PrerequisiteInput(
+        name="illuminationCorrection",
+        doc="Illumination correction frame.",
+        storageClass="Exposure",
+        dimensions=["instrument", "detector", "physical_filter"],
+        isCalibration=True,
+    )
```

Comment:
So, someday when we have a `reference_flux_flat` that is is more sophisticated, is the idea that we'd have a `reference_flux_flat` connection _instead_ of this one, or that this task would keep the `illumination_correction` connection but also get some other input connection(s) (for at least dome flat, I think, and maybe some wavelength-dependent stuff, too)?

Reply:
The answer to this partly depends on where we put the SED-matching of an SED-matched flat.  I don't have an answer to that right now.
But I do believe the input(s) will have to change somewhat, maybe just in an additive fashion.  The output -- the applied `background_to_photometric_ratio` -- I *hope* can stay the same, however.